.circleci/config.yml

```diff
@@ -326,14 +326,14 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - build:
```

Comment:
I believe you'd yell at me that these should be deleted, not commented.

Reply:
Didn't bother since we're probably killing the rest of this shortly but fair enough